src/data/equiv/mul_add_aut.lean

```diff
@@ -130,6 +147,22 @@ add_equiv.apply_symm_apply _ _
 def to_perm : add_aut A →* equiv.perm A :=
 by refine_struct { to_fun := add_equiv.to_equiv }; intros; refl
 
+/-- The tautological action by `add_aut A` on `A`.
+
+This generalizes `function.End.apply_mul_action`. -/
+instance apply_distrib_mul_action {A} [add_monoid A] : distrib_mul_action (add_aut A) A :=
+{ smul := ($),
+  smul_zero := add_equiv.map_zero,
+  smul_add := add_equiv.map_add,
+  one_smul := λ _, rfl,
+  mul_smul := λ _ _ _, rfl }
```

Comment:
I assume this can't be easily `to_additive`d from the `mul_aut` equivalents?

Reply:
There is no multiplicative version of `distrib_mul_action` available (yet) to generalize too, but I believe there should be.